bin/pycbc_compress_bank

```diff
@@ -72,11 +72,17 @@ parser.add_argument("--tolerance", type=float, default=0.001,
                      "added to the compressed waveform until its "
                      "interpolation has a mismatch <= this value. Default is "
                      "0.001.")
-parser.add_argument("--interpolation", type=str, default="linear",
+parser.add_argument("--interpolation", type=str, 
+                default="linear_same_freq_points",
```

Comment:
OK, we'll use the ``None`` method on the second array to do this and throw an error.